docs/02-providers/aws/events/05-streams.md

```diff
@@ -0,0 +1,34 @@
+<!--
+title: DynamoDB / Kinesis Streams configuration docs
+menuText: DynamoDB / Kinesis Streams config
+layout: Doc
+-->
+
+# DynamoDB / Kinesis Streams
+
+This setup specifies that the `compute` function should be triggered whenever the corresponding DynamoDB table is modified (e.g. a new entry is added).
+
+```yml
+functions:
+  compute:
+    handler: handler.compute
+    events:
+      - stream: some:dynamodb:stream:arn
```

Comment:
Add 2 different stream configs, one with an ARN and one that uses the Ref syntax and mention that you can reference a dynamodb or kinesis stream created in the same service through Ref